src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -507,7 +507,7 @@ def load_acs_service_principals(config_path):
 
 def _create_kubernetes(resource_group_name, deployment_name, dns_name_prefix, name, ssh_key_value,
                        admin_username="azureuser", agent_count="3", agent_vm_size="Standard_D2_v2",
-                       location=None, service_principal=None, client_secret=None):
+                       location=None, service_principal=None, client_secret=None, master_count="1"):
```

Comment:
It's always been a string:

https://github.com/Azure/azure-cli/blob/master/src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/mgmt_acs/lib/models/deployment_acs.py#L48